templates/packages/order_edit.html

```diff
@@ -0,0 +1,254 @@
+                                <button type="submit" class="btn btn-outline-success" onclick="submitPackageForm()">אשר</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+        //this is so pretty ^_^
+        document.getElementById('package-modal').addEventListener('show.bs.modal', function (event) {
+            var origin_street = event.relatedTarget.getAttribute('data-bs-origin-street')
+            var origin_street_number = event.relatedTarget.getAttribute('data-bs-origin-street-number')
+            var origin_city = event.relatedTarget.getAttribute('data-bs-origin-city')
+            var origin_area = event.relatedTarget.getAttribute('data-bs-origin-area')
+
+            document.getElementById('origin-street').value = origin_street
+            document.getElementById('origin-street-number').value = origin_street_number
+            document.getElementById('origin-city').value = origin_city
+            document.getElementById('origin-area').value = origin_area
+
+            var destination_street = event.relatedTarget.getAttribute('data-bs-destination-street')
+            var destination_street_number = event.relatedTarget.getAttribute('data-bs-destination-street-number')
+            var destination_city = event.relatedTarget.getAttribute('data-bs-destination-city')
+            var destination_area = event.relatedTarget.getAttribute('data-bs-destination-area')
+
+            document.getElementById('destination-street').value = destination_street
+            document.getElementById('destination-street-number').value = destination_street_number
+            document.getElementById('destination-city').value = destination_city
+            document.getElementById('destination-area').value = destination_area
+
+            var full_name = event.relatedTarget.getAttribute('data-bs-full-name')
+            var phone_number = event.relatedTarget.getAttribute('data-bs-phone-number')
+            var notes = event.relatedTarget.getAttribute('data-bs-notes')
+            var rate = event.relatedTarget.getAttribute('data-bs-rate')
+            document.getElementById('full-name').value = full_name
+            document.getElementById('phone-number').value = phone_number
+            document.getElementById('notes').value = notes
+            document.getElementById('rate').value = rate
+        })
+        function submitPackageForm() {
+            //add validation here
+            document.getElementById('package-form').submit()
+        }
+    </script>
+{%endblock%}
+
+{%block bottom%}
+    {%if has_unsaved_progress%}
+        <form method="post">
+            <input type="hidden" value="save" />
+            <a class="btn btn-outline-success" type="submit">שמור <span class="bi-save"></span></a>
+        </form>
+    {%endif%}
+{%endblock%}
+<datalist id="areas">

```

Comment:
You might want to use a server-side autocomplete. Take a look at [JQuery Autocomplete](https://jqueryui.com/autocomplete/#remote-jsonp)

Reply:
There is an issue opened for this #18